src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -38,47 +46,101 @@
     }
 
     @Test(dataProvider = "gvcfsToGenotype")
-    public void testEntireVariantContext(String input, String expected, List<String> extraArgs) throws IOException {
-        assertVariantContextsMatch(getTestFile(input), getTestFile(expected), extraArgs);
+    public void testEntireVariantContext(File input, File expected, List<String> extraArgs, String reference) throws IOException {
+        assertVariantContextsMatch(input, expected, extraArgs, reference);
     }
 
-    private void assertVariantContextsMatch(File input, File expected, List<String> extraArgs) throws IOException {
+    private void assertVariantContextsMatch(File input, File expected, List<String> extraArgs, String reference) throws IOException {
         runGenotypeGVCFSAndAssertSomething(input, expected, extraArgs, (a, e) -> VariantContextTestUtils.assertVariantContextsAreEqual(a, e,
-                Arrays.asList("FS", //TODO There's a bug in GATK 3 computing FS
-                        "QD", //TODO QD has a cap value and anything that reaches that is randomized.  It's difficult to reproduce the same random numbers accross gatk3 -> 4
-                        "InbreedingCoeff")));
+                Arrays.asList("QD",//TODO QD has a cap value and anything that reaches that is randomized.  It's difficult to reproduce the same random numbers accross gatk3 -> 4
+                              "FS")),//TODO There's some bug in either gatk3 or gatk4 fisherstrand that's making them not agree still, I'm not sure which is correct
+                 reference);
     }
 
-    private void assertGenotypesMatch(File input, File expected, List<String> additionalArguments) throws IOException {
-        runGenotypeGVCFSAndAssertSomething(input, expected, additionalArguments, VariantContextTestUtils::assertVariantContextsHaveSameGenotypes);
+    private void assertGenotypesMatch(File input, File expected, List<String> additionalArguments, String reference) throws IOException {
+        runGenotypeGVCFSAndAssertSomething(input, expected, additionalArguments, VariantContextTestUtils::assertVariantContextsHaveSameGenotypes,
+                                           reference);
     }
 
-    private void runGenotypeGVCFSAndAssertSomething(File input, File expected, List<String> additionalArguments, BiConsumer<VariantContext, VariantContext> assertion) throws IOException {
-        final File output = createTempFile("genotypegvcf", "vcf");
+    private void runGenotypeGVCFSAndAssertSomething(File input, File expected, List<String> additionalArguments, BiConsumer<VariantContext, VariantContext> assertion, String reference) throws IOException {
+        final File output = createTempFile("genotypegvcf", ".vcf");
 
         final ArgumentsBuilder args = new ArgumentsBuilder();
-        args.addReference(new File(b37_reference_20_21))
+        args.addReference(new File(reference))
                 .addVCF(input)
                 .addOutput(output);
 
```

Comment:
There is a TODO below in a comment: "TODO Inbreeding calculation changed between 3.4 and now"  Is this still valid? Did you fix this?